.gitignore

```diff
@@ -20,7 +20,6 @@ htmlcov
 .DS_Store
 package
 dist
-*.zip
```

Comment:
Not sure why this snuck in, but it was causing `INCOMING/MBTA_GTFS.zip` to not be added to the repo